components/http_foundation.rst

```diff
@@ -85,9 +85,9 @@ instance (or a sub-class of), which is a data holder class:
 
 * ``request``: :class:`Symfony\\Component\\HttpFoundation\\ParameterBag`;
```

Comment:
`request` should be too ?

Reply:
Hmm, not all time.... see: https://github.com/symfony/symfony/pull/37265

Any ideas how to document it?

Reply:
Excat, i don't know.. maybe a `note` section ?

Reply:
I don't really understand the attached PR. But could be:

> :class:`Symfony\\Component\\HttpFoundation\\InputBag (when blah, blah) or :class:`Symfony\\Component\\HttpFoundation\\ParameterBag

The "when blah blah" is me showing that, ideally, we would have a short description of when one if used vs the other... but the PR isn't clear to me - maybe it's more clear for some of you.